clang/lib/CIR/CodeGen/CIRGenTypes.cpp

```diff
@@ -202,6 +202,18 @@ mlir::Type CIRGenTypes::convertType(QualType type) {
     break;
   }
 
+  case Type::ConstantArray: {
+    const ConstantArrayType *arrTy = cast<ConstantArrayType>(ty);
+    mlir::Type elemTy = convertTypeForMem(arrTy->getElementType());
+
+    // FIXME: In LLVM, "lower arrays of undefined struct type to arrays of
+    // i8 just to have a concrete type". Not sure this makes sense in CIR yet.
+    assert(builder.isSized(elemTy) && "not implemented");
```

Comment:
This should probably use the `cgm.errorNYI`, then just come up with a way to recover.  That said, this is exactly why having `isSized` assert is awkward here.

Reply:
Okey, thats make sense and i will try to recover with 0 size